lab_solutions/lab02_12_are_anagrams.py

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2020 Bence Bial <gbenceg@gmail.com>
+#
+# Distributed under terms of the MIT license.
+
+from lab_solutions.lab02_04_char_freq import char_freq
+
+
+def are_anagrams(word1, word2):
+    return char_freq(word1) == char_freq(word2)
+
+
+def main():
+    assert (are_anagrams("abc", "bac") == True)
```

Comment:
No need for the parentheses since assert is a statement, not a function:
```suggestion
    assert are_anagrams("abc", "bac") == True
```

Reply:
Assert statements were copied from each given task. Btw PEP8 suggests to use the following format:
`assert are_anagrams(...)`
`assert not are_anagrams(...)`

Reply:
Thank you for pointing that out. I'll fix it next time.